src/addon.cc

```diff
@@ -190,21 +189,20 @@ void handleReceived_Data(Isolate *isolate, SIMCONNECT_RECV *pData, DWORD cbData)
 	for (int i = 0; i < numVars; i++)
 	{
 		int varSize = 0;
+		v8::Local<v8::String> key = String::NewFromUtf8(isolate, valIds.at(i).c_str());
+		char *pData = ((char*)(&pObjData->dwData) + dataValueOffset);
 
 		if (valTypes[i] == SIMCONNECT_DATATYPE_STRINGV)
 		{
-			dataValueOffset += 8; // Not really sure why this is needed, but it fixes problems like this: "F-22 RapF-22 Raptor - 525th Fighter Squadron"
```

Comment:
Have you verified that you get the expected output string? I still wonder why I had to add this line 🤔

Reply:
At least in FS2020 it was eating up the beginning of the string. I couldn't find a reason to keep it, so it might be good to try this version on the sim you tried it on first

Reply:
Looks like it's no longer needed after the changes you made. Just tested with FSX. Good job 👍